packages/components/src/internal/components/lineage/node/LineageNodeDetail.tsx

```diff
@@ -185,14 +185,17 @@ const RunStepNodeDetail: FC<RunStepNodeDetailProps> = memo(props => {
         <div className="run-step-node-detail">
             <DetailHeader header={`Run Step: ${stepName}`} iconSrc="default">
                 <a className="lineage-link" onClick={onBack}>
-                    {node.name}
+                    <span>Back to Run Details</span>
                 </a>
                 <span className="spacer-left">&gt;</span>
                 <span className="spacer-left">{stepName}</span>
             </DetailHeader>
             <Tabs activeKey={tabKey} defaultActiveKey={1} id="lineage-run-step-tabs" onSelect={changeTab as any}>
                 <Tab eventKey={1} title="Step Details">
                     <LineageDetail item={step} />
+                    {step.properties.length > 0 && (
```

Comment:
will the `step` object always have `properties` defined? or should this have a null check like `step?.properties.length`?